applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -173,6 +248,54 @@ static uint32_t get_and_incr_seq_num(const struct bt_audio_stream *stream)
 	return 0;
 }
 
+static int update_sink_stream_qos(uint32_t pres_delay_us)
+{
+	int ret;
+
+	lc3_preset_sink.qos.pd = pres_delay_us + 10;
```

Comment:
What is this +10?

Reply:
There are some other magic numbers here as well.

Reply:
These were for testing and have been removed.